README.md

```diff
@@ -112,28 +114,50 @@ wire.ut_pex.reset()
 
 ### addPeer
 
-Adds a peer to the locally discovered peer list to send with the next PEX message.
+Adds an IPv4 peer to the locally discovered peer list to send with the next PEX message.
 
 ```js
-wire.ut_pex.addPeer('127.0.0.1:6889')
+const peer = '127.0.0.1:6889'
+const flags = 0x06
+
+wire.ut_pex.addPeer(peer, flags)
+```
+
+### addPeer6
+
+Adds an IPv6 peer to the locally discovered peer list to send with the next PEX message.
+
+```js
+const peer = '[::1]:6889'
+const flags = 0x06
```

Comment:
should we make an API for the accepted flags? Like we could use function arguments like `supports_utp` `prefers_encryption` etc and then set them as properties on the peer object like `peer.supports_utp === true`

we should decide now before cementing an API

Reply:
actually, not sure if I fully understand the BEP, but flags should be a list not just one number I think?

Reply:
Yeah! I think it is a good idea handle the flags as properties 👍 I'll try to work on it a bit 😉

Reply:
I've added an API for flags, hope it is more clear now 😉